showcases/doc/showcases.md

```diff
@@ -26,7 +26,6 @@ We need the following styles for the `box` demos:
     height: 6rem;
     box-shadow: 0 25px 50px -12px rgba(0, 0, 0, 0.25);
     background-color: #2f855a;
-    border-radius: 0.2rem;
```

Comment:
as mentioned in the previous PR, border radius + checkered bg doesn't work well, as the border radius is not applied to the checkered bg. So for showcasing bg color with opacity, where the checkered bg is needed, boder radius was removed.

Reply:
IIRC Border-radius does not apply with background, you need to constraint the box with overflow hidden and then the border-radius should be visible